doc/metrics.md

```diff
@@ -0,0 +1,320 @@
+```
+
+
+## SAP Enqueue Server
+
+The Enqueue Server (also known as the lock server) is the SAP system component that manages the lock table.
+
+01. [`sap_enqueue_server_arguments_high`](#sap_enqueue_server_arguments_high)
+02. [`sap_enqueue_server_arguments_max`](#sap_enqueue_server_arguments_max)
+03. [`sap_enqueue_server_arguments_now`](#sap_enqueue_server_arguments_now)
+04. [`sap_enqueue_server_arguments_state`](#sap_enqueue_server_arguments_state)
+05. [`sap_enqueue_server_backup_requests`](#sap_enqueue_server_backup_requests)
+06. [`sap_enqueue_server_cleanup_requests`](#sap_enqueue_server_cleanup_requests)
+07. [`sap_enqueue_server_compress_requests`](#sap_enqueue_server_compress_requests)
+08. [`sap_enqueue_server_dequeue_all_requests`](#sap_enqueue_server_dequeue_all_requests)
+09. [`sap_enqueue_server_dequeue_errors`](#sap_enqueue_server_dequeue_errors)
+10. [`sap_enqueue_server_dequeue_requests`](#sap_enqueue_server_dequeue_requests)
+11. [`sap_enqueue_server_enqueue_errors`](#sap_enqueue_server_enqueue_errors)
+12. [`sap_enqueue_server_enqueue_rejects`](#sap_enqueue_server_enqueue_rejects)
+13. [`sap_enqueue_server_enqueue_requests`](#sap_enqueue_server_enqueue_requests)
+14. [`sap_enqueue_server_lock_time`](#sap_enqueue_server_lock_time)
+15. [`sap_enqueue_server_lock_wait_time`](#sap_enqueue_server_lock_wait_time)
+16. [`sap_enqueue_server_locks_high`](#sap_enqueue_server_locks_high)
+17. [`sap_enqueue_server_locks_max`](#sap_enqueue_server_locks_max)
+18. [`sap_enqueue_server_locks_now`](#sap_enqueue_server_locks_now)
+19. [`sap_enqueue_server_locks_state`](#sap_enqueue_server_locks_state)
+20. [`sap_enqueue_server_owner_high`](#sap_enqueue_server_owner_high)
+21. [`sap_enqueue_server_owner_max`](#sap_enqueue_server_owner_max)
+22. [`sap_enqueue_server_owner_now`](#sap_enqueue_server_owner_now)
+23. [`sap_enqueue_server_owner_state`](#sap_enqueue_server_owner_state)
+24. [`sap_enqueue_server_replication_state`](#sap_enqueue_server_replication_state)
+25. [`sap_enqueue_server_reporting_requests`](#sap_enqueue_server_reporting_requests)
+26. [`sap_enqueue_server_server_time`](#sap_enqueue_server_server_time)
+27. [`sap_enqueue_server_verify_requests`](#sap_enqueue_server_verify_requests)
+
+### `sap_enqueue_server_arguments_high`
+
+#### Example
+
+```
+# TYPE sap_enqueue_server_arguments_high gauge
+sap_enqueue_server_arguments_high 104
+```
+
+### `sap_enqueue_server_arguments_max`
+
+#### Example
+
+```
+# TYPE sap_enqueue_server_arguments_max gauge
+sap_enqueue_server_arguments_max 56415
+```
+
+### `sap_enqueue_server_arguments_now`
+
+#### Example
+
+```
+# TYPE sap_enqueue_server_arguments_now gauge
+sap_enqueue_server_arguments_now 0
+```
+
+### `sap_enqueue_server_arguments_state`
+
+#### Example
+
+```
+# TYPE sap_enqueue_server_arguments_state gauge
+sap_enqueue_server_arguments_state 2
+```
+
+### `sap_enqueue_server_backup_requests`
```

Comment:
we might open a follow-up issue to document better this

Reply:
yeah, I need to dig through the SAP documentation further, to understand how to describe all these metrics.